documentation/asciidoc/topics/proc_creating_service_monitor.adoc

```diff
@@ -1,53 +1,26 @@
 [id='creating-service-monitor_{context}']
 = Creating a Prometheus Service Monitor
-Create a Prometheus `ServiceMonitor` that scrapes metrics from a {brandname} cluster.
+{ispn_operator} automatically creates a Prometheus `ServiceMonitor` that scrapes metrics from your {brandname} cluster.
 
-.Prerequisites
+.Procedure
 
-* Have an `oc` client.
-* Enable monitoring for user-defined projects on {ocp}.
+Enable monitoring for user-defined projects on {ocp} so that the `ServiceMonitor` resource is available on the {openshiftshort} cluster.
 
-.Procedure
+When it detects the `ServiceMonitor` resource, {ispn_operator} does the following:
 
-. Create an authentication secret for the `operator` user.
-+
-Prometheus needs credentials to authenticate with {brandname} and must use the `operator` user and corresponding password.
-+
-.. Retrieve the `operator` user credentials.
-+
-[source,options="nowrap",subs=attributes+]
-----
-include::cmd_examples/oc_get_secret_operator_creds.adoc[]
-----
-+
-.. Create an authentication secret, for example:
-+
-[source,options="nowrap",subs=attributes+]
-----
-include::yaml/basic_auth.yaml[]
-----
-+
-.. Add the authentication secret to your {brandname} cluster namespace.
-+
-[source,options="nowrap",subs=attributes+]
-----
-$ oc apply -f basic-auth.yaml
-----
-+
-. Create a Prometheus `ServiceMonitor` that scrapes {brandname} metrics.
-+
-[source,options="nowrap",subs=attributes+]
-----
-include::yaml/prometheus_service_monitor.yaml[]
-----
-+
-. Add the `ServiceMonitor`.
-+
-[source,options="nowrap",subs=attributes+]
+* Creates a `ServiceMonitor` named `<cluster_name>-monitor`.
+* Adds the `infinispan.org/monitoring: 'true'` annotation to your `Infinispan` CR.
+
+[source,yaml,options="nowrap",subs=attributes+]
 ----
-$ oc apply -f service-monitor.yaml
+include::yaml/infinispan-monitoring-true.yaml[]
 ----
 
+[NOTE]
+====
+To authenticate with {brandname}, Prometheus uses the `operator` credentials.
```

Comment:
@ryanemerson do we need to document how to change operator admin password and that `password` filed is primary truth entry?

Reply:
I tried pushing the following commit to this PR, but it was rejected. Here's the docs I wrote on Friday:

https://github.com/infinispan/infinispan-operator/compare/master...ryanemerson:password_doc_updates?expand=1

Reply:
@dmvolod and @ryanemerson I can add those docs to this PR. weird that that commit was rejected.

Reply:
Thanks @oraNod and @ryanemerson